plugins/modules/nd_federation_member.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+import base64
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_federation_member
+version_added: "0.3.0"
+short_description:
+    - Setup multi-cluster configuration on Cisco Nexus Dashboard (ND).
+description:
+    - Connects to another Nexus Dashboard (ND) cluster for a single pane of glass view into all clusters’ sites and services.
+    - M(cisco.nd.nd_federation_member) can only be used with python 3.7 and higher.
+author:
+    - Anvitha Jain (@anvjain)
+options:
+  cluster:
```

Comment:
Have you considered exposing this as list of members input?

Reply:
It only lets me add one. so based on current UI flow, cluster is not exposed as list.